specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-07-09-preview/examples/ManagedCCF_List.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
note: i would not update MCCF RP

Reply:
Didnt make change for mccf. I just copy the whole payload from last version.